pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -231,7 +231,9 @@ func (m *ManagerImpl) Start(activePods ActivePodsFunc, sourcesReady config.Sourc
 	}
 
 	socketPath := filepath.Join(m.socketdir, m.socketname)
-	os.MkdirAll(m.socketdir, 0750)
+	if err = os.MkdirAll(m.socketdir, 0750); err != nil {
```

Comment:
Adding error checking here works for me. From the [Golang docs](https://golang.org/pkg/os/#MkdirAll), its clear that `MkdirAll` will not raise an err if the dir already exists.